### PR TITLE
Con 2330 manage groups back end work

### PR DESCRIPTION
**Pull request by karpagamNallasamy**
Manual Pretesting completed by Prasath. in my local branch.

### CODE REVIEW

File: api/CcsSso.Core.Domain/Constants/Constants.cs [hunk 1]
```diff
@@ -46,6 +46,7 @@ public static class ErrorConstant
     public const string ErrorCannotDeleteLastOrgAdmin = "ERROR_CANNOT_DELETE_LAST_ADMIN_OF_ORGANISATION";
     public const string ErrorCannotRemoveAdminRoleGroupLastOrgAdmin = "ERROR_CANNOT_REMOVE_ADMIN_ROLE_OR_GROUP_OF_LAST_ADMIN";
     public const string ErrorInvalidGroupName = "INVALID_GROUP_NAME";
+    public const string ErrordonotallowallspecialcharacterGroupName = "ENTER_YOUR_GROUP_NAME_WITHOUT_ALL_SPECIAL_CHARACTERS";
```
Comment (VijayHirudayasamy-bc) [inline api/CcsSso.Core.Domain/Constants/Constants.cs:49]: Minor one. for readability point of view can you use the variable like this please ErrorDonotAllowAllSpecialCharacterGroupName

File: api/CcsSso.Core.Service/External/OrganisationGroupService.cs [hunk 2]
```diff
@@ -73,6 +81,23 @@ public async Task<int> CreateGroupAsync(string ciiOrganisationId, OrganisationGr
       return group.Id;
     }
 
+    private bool ValidateGroupName(string str)
```
Comment (VijayHirudayasamy-bc) [inline api/CcsSso.Core.Service/External/OrganisationGroupService.cs:84]: Is it used anywhere in the code. if we are not using then remove it please

**Review by VijayHirudayasamy-bc — CHANGES_REQUESTED** [review]
I have minor comments. Can you make those changes pls

**Comment by karpagamNallasamy** [comment]
@Vijay ***@***.***> requested changes on my PR committed . please review and merge.

________________________________

**Review by VijayHirudayasamy-bc — APPROVED** [review]
Hope you have done QA buddy check

**Pull request merged**